src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -254,6 +256,37 @@ public bool NotUsingHttps(string url)
             }
         }
 
+        [RavenAction("/admin/databases/reorder","POST",AuthorizationStatus.Operator)]
+        public async Task Reorder()
+        {
+            var name = GetStringQueryString("name");
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            {
+                var record = ServerStore.LoadDatabaseRecord(name, out var _);
+                if (record == null)
+                {
+                    DatabaseDoesNotExistException.Throw(name);
+                }
+                var json = await context.ReadForMemoryAsync(RequestBodyStream(), "nodes");
+                var parameters = JsonDeserializationServer.Parameters.MembersOrder(json);
+
+                if (record.Topology.Members.All(parameters.MembersOrder.Contains) == false)
```

Comment:
That is just one way to check, also need to check the other directions. `MemberOrder` that isn't on the `Topology.Members`